test/cmdlineTests.sh

```diff
@@ -117,6 +117,7 @@ test_solc_behaviour() {
     local stdout_expected="${4}"
     local exit_code_expected="${5}"
     local stderr_expected="${6}"
+    local standard_json="${7}"
```

Comment:
This is getting really messy, but let's clean up as part of the cleanup issue that already exists.

Reply:
Yes... I should probably have grepped for ``--standard-json`` in the arguments instead.